examples/COCO_people_subset.ipynb

```diff
@@ -149,20 +150,23 @@
     "                poly = []\n",
     "                for s in seg:\n",
     "                    poly_arr = np.array(s).reshape(-1,2)\n",
-    "                    poly += [[poly_arr[i,0]/width, poly_arr[i,1]/height] for i in range(len(poly_arr))]\n",
+    "                    poly += [(poly_arr[i,0]/width, poly_arr[i,1]/height) for i in range(len(poly_arr))]\n",
     "                yield {\n",
     "                    \"file\": path,\n",
     "                    \"class\": \"person\",\n",
     "                    \"type\": \"polyline\",\n",
     "                    \"value\": poly\n",
     "                }\n",
     "            else: # RLE format\n",
-    "                rle = (seg[\"size\"][0], seg[\"size\"][1], seg[\"counts\"])\n",
+    "                height, width = seg[\"size\"]\n",
+    "                rle = mask_utils.frPyObjects(seg, height, width)\n",
+    "\n",
+    "                value = (rle[\"size\"][0], rle[\"size\"][1], list(rle[\"counts\"]))\n",
```

Comment:
In this example we either need to restore to what it was or not cast to `list` since the `rle["counts"]` are `bytes` here